myfitnesspal/client.py

```diff
@@ -1243,13 +1226,15 @@ def get_recipe(self, recipeid: int) -> dict:
         )[0].text.strip(" \n")
 
         # add some required tags to match schema
-        recipe_dict["recipe_instructions"] = []
+        recipe_dict["recipeInstructions"] = ""
```

Comment:
Looking at the schema for this, I believe this is actually the right key name.

Reply:
You are right. 👍